services/state/token/token_relation.proto

```diff
@@ -0,0 +1,73 @@
+syntax = "proto3";
+
+package proto;
+
+/*-
+ * ‌
+ * Hedera Network Services Protobuf
+ * ​
+ * Copyright (C) 2018 - 2023 Hedera Hashgraph, LLC
+ * ​
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * ‍
+ */
+
+import "basic_types.proto";
+
+option java_package = "com.hederahashgraph.api.proto.java";
+// <<<pbj.java_package = "com.hedera.hapi.node.state.token">>> This comment is special code for setting PBJ Compiler java package
+option java_multiple_files = true;
+/**
+ * First-draft representation of a Hedera Token Service token relationship entity in the network Merkle tree.
+ *
+ * As with all network entities, a token relationship has a unique entity number pair, which is represented
+ * with the account and the token involved in the relationship.
+ */
+message TokenRelation {
```

Comment:
Is this related to the existing TokenRelationship entity?

Reply:
yes, `MerkleTokenRelStatus`

Reply:
I guess what I'm asking, is if we're duplicating structure here, changing existing entities, or creating a new and unrelated entity for internal use?
If we're duplicating the entity, we need to be careful about unsynchronized changes to the two and should probably document in both that they're related and need to be kept in sync.
If we're changing the existing entity, we probably need to document that carefully and deprecate the other definition.
If we're creating a new (more-or-less) unrelated entity, then it should probably be documented here that this isn't a replacement or compliment to that entity.

Reply:
I think the idea is we will not have `Merkle*` entities after refactor. We will be using the PBJ types in all Maps in state.

Reply:
Thanks.  So this is replacing internal data structures, not replicating or replacing external entities.  That makes sense, and helps to understand where these fit.